src/commands/mobile/plugin/sentry.ts

```diff
@@ -0,0 +1,167 @@
+    const projectRoot = path.join(process.cwd()).trim();
+
+    if (!projectName) {
+      const rootArray = projectRoot.split('\\');
+      projectName = rootArray[rootArray.length - 1];
+    }
+
+    // block command unless being run within an rdvue project
+    if (isValidProject === false && !hasProjectName) {
+      throw new Error(
+        JSON.stringify({
+          code: 'project-invalid',
+          message: `${MOBILE_CLI_COMMANDS.PluginSentry} command must be run in an existing ${chalk.yellow('rdvue')} project`,
+        }),
+      );
+    }
+
+    const folderList = TEMPLATE_FOLDERS;
+    const organization = await parseOrganization(args);
+    const authToken = await parseAuthorizationToken(args);
+
+    // parse config files required for scaffolding this module
+    const configs = parseMobileModuleConfig(folderList, projectRoot);
+    const config = configs[0];
+    const files: Array<string | Files> = config.manifest.files;
+    const dependencies = config.manifest.packages.dependencies.toString()
+      .split(',')
+      .join(' ');
+
+    if (skipInstallStep === false) {
+      try {
+        // install dependencies
+        ux.action.start(`${CLI_STATE.Info} installing sentry dependencies`);
+        await exec(`${preInstallCommand} npm install --save ${dependencies} --legacy-peer-deps`, { silent: true });
+        ux.action.stop();
+      } catch (error) {
+        throw new Error(
+          JSON.stringify({
+            code: 'dependency-install-error',
+            message: `${this.id?.split(':')[1]} sentry dependencies failed to install`,
+          }),
+        );
+      }
+    } else {
+      ux.action.start(`${CLI_STATE.Info} adding sentry dependencies`);
+      await exec(`cd ${projectName} && npx add-dependencies ${dependencies} --legacy-peer-deps`, { silent: true });
+      ux.action.stop();
+    }
+
+    const sourceDirectory: string = path.join(config.moduleTemplatePath, config.manifest.sourceDirectory);
+    const installDirectory: string = path.join(projectRoot, 'src/core', config.manifest.installDirectory);
+
+    // copy and update files for plugin being added
+    await copyFiles(sourceDirectory, installDirectory, files);
+
+    // update imports
+    updateDynamicImportsAndExports(projectRoot, `core/${config.manifest.installDirectory}`, config.manifest.moduleImports, 'index.ts');
+
+    // update app.json file
+    const appConfigurationLocation = path.join(projectRoot, 'app.json');
+    const appConfiguration = readConfigFile(appConfigurationLocation, 'app.json file is not found.');
+
+    if (appConfiguration.expo.plugins !== undefined && typeof appConfiguration.expo.plugins === 'object')  {
+      appConfiguration.expo.plugins.push('sentry-expo');
+    } else {
+      appConfiguration.expo.plugins = ['sentry-expo'];
+    }
+
+    const expoPostPublishConfiguration = {
+      file: 'sentry-expo/upload-sourcemaps',
+      config: {
+        project: projectName,
```

Comment:
Should this `project` property by the absolute path or should it be the name of the project only?

Reply:
Currently, it is the absolute path when the plugin is added.

Reply:
@ttaylor92